frontend/src/bundles/profile/store/actions.ts

```diff
@@ -1,24 +1,27 @@
 import { createAsyncThunk } from '@reduxjs/toolkit';
-import { type SocialMediaProfiles } from 'shared/build';
+import {
+    type Profile,
+    type SocialMediaProfiles,
+    type UpdateUserProfileAndEmailRequestDto,
+} from 'shared/build';
 
+import { actions } from '~/bundles/auth/store/slice.js';
```

Comment:
```suggestion
import { actions as authActions } from '~/bundles/auth/store/slice.js';
```
Maybe say explicitly that these are authActions.

Reply:
done